src/components/PatchesPage/index.tsx

```diff
@@ -76,27 +72,29 @@ export const PatchesPage: React.VFC<Props> = ({
     <PageWrapper>
       <PageTitle data-cy="patches-page-title">{pageTitle}</PageTitle>
       <FiltersWrapperSpaceBetween>
-        <FlexRow>
-          <StyledInput
-            placeholder="Search Patch Descriptions"
-            onChange={(e) => setAndSubmitInputValue(e.target.value)}
-            suffix={<Icon glyph="MagnifyingGlass" />}
-            value={inputValue}
-            data-cy="patch-description-input"
-            width="25%"
-          />
-          <StatusSelector />
-        </FlexRow>
-        <Checkbox
-          data-cy="commit-queue-checkbox"
-          onChange={onCheckboxChange}
-          label={
-            pageType === "project"
-              ? "Only Show Commit Queue Patches"
-              : "Include Commit Queue"
-          }
-          checked={isCommitQueueCheckboxChecked}
+        <TextInputWithGlyph
+          glyph="MagnifyingGlass"
+          label=""
+          description=""
```

Comment:
I think you can omit `label` and `description`, but you will need to add an `aria-label` props to get rid of the TS error that results from that